views/pages/register.handlebars

```diff
@@ -0,0 +1,55 @@
+<div class="row">
+    <div class="col-sm-12">
+        <form id="register-form" method="post" action="/users/create">
+            <div class="form-group">
+                <label for="userName">UserName</label>
+                <input type="text" class="form-control" id="userName" name="username" placeholder="userName"
+                    value="{{userName}}" required>
+            </div>
+            <div class="form-group">
+                <label for="email">Email</label>
+                <input type="email" class="form-control" id="email" name="email" placeholder="email" value="{{email}}"
+                    required>
+            </div>
+            <div class="form-group">
+                <label for="password">Password</label>
+                <input type="password" class="form-control" id="password" name="password" placeholder="something.."
+                    value="{{password}}" required>
+            </div>
+            <div class="form-group">
+                <label for="nickName">Handle</label>
+                <input type="text" class="form-control" id="nickName" name="nickName" placeholder="nickName"
+                    value="{{nickName}}" required>

```

Comment:
`            <div class="form-group">
                <label for="nickName">Handle</label>
                <input type="text" class="form-control" id="nickName" name="nickName" placeholder="nickName"
                    value="{{nickName}}" required>`

I don't think nickname should be there since we already have userName. I think first name and last name would be better here